backend/config/.env.example

```
@@ -24,6 +24,7 @@ REDIS_PORT=6379
 REDIS_DB=0
 # REDIS_PASSWORD=your-secure-password  # Uncomment and set for production
 # REDIS_USERNAME=default  # Uncomment if using Redis 6.0+ ACL
+REDIS_SSL=false  # Set to true for TLS (e.g. AWS ElastiCache with transit encryption)
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)